#6 Added possibility to provide custom guzzle options

duxthefux opened the pull request:
This PR adds the possibility to provide custom guzzle options when using `ClientBuilder`.

This is essential if you have custom authorization methods, need to set cookies, headers, etc.

rccrdpccl commented on src/ClientBuilder.php:
```diff
@@ -7,23 +7,40 @@
-            new \Softonic\GraphQL\ResponseBuilder()
-        );
-    }
+	/**
```
This comment does not add anything to the type hint already in place.

rccrdpccl commented on src/ClientBuilder.php:
```diff
@@ -7,23 +7,40 @@
+	 */
+	public static function build(string $endpoint, array $guzzleOptions = []) : Client
+	{
+		// allow custom guzzle options
```
This comment does not add anything to the already well-named variable.

duxthefux commented:
hey, I've added two tests to run the ClientBuilder with $guzzleOptions (i used  "cookies" - option)
i also removed phpdocs again, as you are right, as it's redundant.

Kind regards

rccrdpccl commented:
Thanks!

duxthefux commented:
thanks for merging :-) would you mind creating a new release so I can use it in composer.json?

Thanks!

rccrdpccl commented:
Sure: https://github.com/softonic/graphql-client/releases/tag/1.1.0
